WordPressKit/WordPressKit/AccountServiceRemoteREST.m

```diff
@@ -26,22 +26,15 @@ - (void)getBlogsWithSuccess:(void (^)(NSArray *))success
 {
     NSString *requestUrl = [self pathForEndpoint:@"me/sites"
                                      withVersion:ServiceRemoteWordPressComRESTApiVersion_1_1];
+    [self getBlogsWithRequestURL:requestUrl success:success failure:failure];
+}
 
-    NSString *locale = [[WordPressComLanguageDatabase new] deviceLanguageSlug];
-    NSDictionary *parameters = @{
-                                 @"locale": locale
-                                 };
-    [self.wordPressComRestApi GET:requestUrl
-       parameters:parameters
-                    success:^(id responseObject, NSHTTPURLResponse *httpResponse) {
-              if (success) {
-                  success([self remoteBlogsFromJSONArray:responseObject[@"sites"]]);
-              }
-          } failure:^(NSError *error, NSHTTPURLResponse *httpResponse) {
-              if (failure) {
-                  failure(error);
-              }
-          }];
+- (void)getVisibleBlogsWithSuccess:(void (^)(NSArray *))success
+                           failure:(void (^)(NSError *))failure
+{
+    NSString *requestUrl = [self pathForEndpoint:@"me/sites?site_visibility=visible"
+                                     withVersion:ServiceRemoteWordPressComRESTApiVersion_1_1];
+    [self getBlogsWithRequestURL:requestUrl success:success failure:failure];
```

Comment:
Maybe this call should support a parameters objects, instead of hacking the endpoint directly?